src/main/kotlin/com/terraformation/backend/customer/model/IndividualUser.kt

```diff
@@ -1,10 +1,10 @@
 package com.terraformation.backend.customer.model
 
-import com.terraformation.backend.auth.CurrentUserHolder
 import com.terraformation.backend.auth.SuperAdminAuthority
 import com.terraformation.backend.auth.currentUser
 import com.terraformation.backend.customer.db.ParentStore
 import com.terraformation.backend.customer.db.PermissionStore
+import com.terraformation.backend.customer.model.TerrawareUser.Companion.log
```

Comment:
This will cause log messages to say they come from `TerrawareUser` rather than `IndividualUser`.

But also, are we logging anything in this class any more? Maybe this isn't needed at all.

Reply:
To answer you question of if we're logging anything anymore, I left the log inside of `getPassword`. 
I'll create a separate companion object inside `IndividualUser` so it doesn't say `TerrawareUser`, but as the code is now, both `IndividualUser` and `TerrawareUser` need it which is why I left it in